src/go/rpk/pkg/config/config_test.go

```diff
@@ -1741,6 +1747,16 @@ func TestReadFlat(t *testing.T) {
 		},
 	}}
 
+	conf.Redpanda.KafkaApiTLS = []ServerTLS{{
+		Name:     "internal",
+		KeyFile:  "/some/key/file.pem",
+		CertFile: "some/cert/file.crt",
+	}, {
+		Name:     "external",
+		KeyFile:  "/some/other/file.pem",
+		CertFile: "some/other/cert.crt",
```

Comment:
Can one of these have the complete set of fields; `enabled`, `truststore_file`, `require_client_auth`